**Linux with security manager** [Build 3237](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=74180&buildTypeId=WF_PullRequest_LinuxSm) outcome was **FAILURE** using a merge of d2d6751abcbd91cd161460de383cb8dc0bf16af4
Summary: Tests failed: 17 (17 new), passed: 923, ignored: 297 Build time: 00:07:13
### Failed tests
```
org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneEC2FullHA: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase8376577113064707407xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneEC2FullHA(StandardConfigsXMLValidationUnitTestCase.java:123)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHostMaster: java.lang.AssertionError: 1:72 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase2823789804262401262xml' cvc-elt.1.a: Cannot find the declaration of element 'host'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHostMaster(StandardConfigsXMLValidationUnitTestCase.java:87)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandalone: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase1342621522212673801xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandalone(StandardConfigsXMLValidationUnitTestCase.java:97)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHost: java.lang.AssertionError: 1:72 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase3775679097993480388xml' cvc-elt.1.a: Cannot find the declaration of element 'host'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHost(StandardConfigsXMLValidationUnitTestCase.java:77)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneHA: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase5364148632930058280xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneHA(StandardConfigsXMLValidationUnitTestCase.java:102)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneJTS: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase4222943507038834172xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneJTS(StandardConfigsXMLValidationUnitTestCase.java:138)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneRTS: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase6895127160813498360xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneRTS(StandardConfigsXMLValidationUnitTestCase.java:158)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneXTS: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase5154790371695631242xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneXTS(StandardConfigsXMLValidationUnitTestCase.java:153)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneGossipHA: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase2183574631503899489xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandaloneGossipHA(StandardConfigsXMLValidationUnitTestCase.java:128)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHostSlave: java.lang.AssertionError: 1:58 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase3059922506974909801xml' cvc-elt.1.a: Cannot find the declaration of element 'host'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testHostSlave(StandardConfigsXMLValidationUnitTestCase.java:82)


org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandalonePicketLink: java.lang.AssertionError: 1:60 systemId='file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/smoke/target/StandardConfigsXMLValidationUnitTestCase7036457899119253372xml' cvc-elt.1.a: Cannot find the declaration of element 'server'. a possible cause may be that a subsystem is not using the most up to date schema.
	at org.jboss.as.test.smoke.subsystem.xml.AbstractValidationUnitTest$ErrorHandlerImpl.error(AbstractValidationUnitTest.java:288)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:394)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:282)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2143)
	at org.apache.xerces.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:788)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:283)
	at org.apache.xerces.impl.XMLNSDocumentScannerImpl$NSContentDispatcher.scanRootElementHook(XMLNSDocumentScannerImpl.java:733)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(XMLDocumentFragmentScannerImpl.java:1754)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:324)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:875)
	at org.apache.xerces.parsers.XML11Configuration.parse(XML11Configuration.java:798)
	at org.apache.xerces.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:186)
	at org.apache.xerces.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:129)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.parseXml(StandardConfigsXMLValidationUnitTestCase.java:175)
	at org.jboss.as.test.smoke.subsystem.xml.StandardConfigsXMLValidationUnitTestCase.testStandalonePicketLink(StandardConfigsXMLValidationUnitTestCase.java:148)



##### there are 7 more failed tests, see build details
```

--- a/pom.xml
+++ b/pom.xml
@@ -214,7 +214,7 @@
         <version.org.scannotation>1.0.3</version.org.scannotation>
         <version.org.slf4j>1.7.7.jbossorg-1</version.org.slf4j>
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
-        <version.org.wildfly.build-tools>1.1.0.CR1</version.org.wildfly.build-tools>
+        <version.org.wildfly.build-tools>1.1.0.CR2</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.4.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.core>2.0.0.CR6</version.org.wildfly.core>
         <version.org.wildfly.arquillian>1.0.1.Final</version.org.wildfly.arquillian>

--- a/servlet-dist/server-provisioning.xml
+++ b/servlet-dist/server-provisioning.xml
@@ -19,7 +19,7 @@
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
-<server-provisioning xmlns="urn:wildfly:server-provisioning:1.0" extract-schemas="true" copy-module-artifacts="true">
+<server-provisioning xmlns="urn:wildfly:server-provisioning:1.1" extract-schemas="true" copy-module-artifacts="true" extract-schemas-groups="org.jboss.as org.wildfly org.wildfly.core">
     <feature-packs>
         <feature-pack groupId="org.wildfly" artifactId="wildfly-servlet-feature-pack" version="${project.version}"/>
     </feature-packs>

--- a/testsuite/mixed-domain/pom.xml
+++ b/testsuite/mixed-domain/pom.xml
@@ -27,8 +27,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <artifactId>wildfly-testsuite</artifactId>
         <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-testsuite</artifactId>
         <version>10.0.0.CR3-SNAPSHOT</version>
     </parent>
 
@@ -67,6 +67,17 @@
         <dependency>
             <groupId>org.wildfly.core</groupId>
             <artifactId>wildfly-subsystem-test-framework</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wildfly</groupId>
+            <artifactId>wildfly-testsuite-shared</artifactId>
+            <exclusions>
+                <!-- exclude woodstox as it is doesn't work right with adding new lines after document end -->
+                <exclusion>
+                    <groupId>org.codehaus.woodstox</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
     </dependencies>
     <build>

